immich/upload.py

```
@@ -0,0 +1,364 @@
+                )
+                if response.status_code == 201:
+                    uploaded.append(
+                        UploadedEntry(asset=response.data, filepath=filepath)
+                    )
+                elif response.status_code == 200:
+                    rejected.append(
+                        RejectedEntry(filepath=filepath, asset_id=response.data.id)
+                    )
+                if not dry_run:
+                    pbar.update(filepath.stat().st_size)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)